.github/workflows/ci.yml

```diff
@@ -0,0 +1,127 @@
+name: ci
+
+on: [push, pull_request]
+
+env:
+  NODE_VERSION: 10
+  SOLR_VERSION: 6.6.5
+
+jobs:
+
+  build-static:
+    name: Build static files
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Setup node
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+    - name: Cache node modules
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: npm-${{ hashFiles('package-lock.json') }}
+        restore-keys: |
+          npm-${{ hashFiles('package-lock.json') }}
+          npm-
+    - name: Install dependencies
+      run: npm ci
+    - name: Run webpack build
+      run: npm run build:prod
+    - name: Upload built files
+      uses: actions/upload-artifact@v2
+      with:
+        name: bundles
+        path: bundles/
+    - name: Upload stats file
+      uses: actions/upload-artifact@v2
+      with:
+        name: webpack-stats.json
+        path: webpack-stats.json
+
+  ts-unit:
+    name: TypeScript unit tests
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Setup node
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+    - name: Cache node modules
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: npm-${{ hashFiles('package-lock.json') }}
+        restore-keys: |
+          npm-${{ hashFiles('package-lock.json') }}
+          npm-
+    - name: Install dependencies
+      run: npm ci
+    - name: Run Jest
+      run: npx jest --collectCoverage
+    - name: Upload test coverage to Codecov
+      uses: codecov/codecov-action@v1
+      with:
+        flags: typescript
+
+  py-unit:
+    name: Python unit tests
+    runs-on: ubuntu-latest
+    needs: build-static
+    strategy:
+      matrix:
+        python: [3.5, 3.6, 3.7]
```

Comment:
It's great that we can do a python build matrix with GitHub actions, but we don't need that for the webapps — only the python libraries that we want to be usable with multiple versions of python. For webapps, we should just target the version of python that the application will run under in production. (I forget if mep is still on 3.5 or if it's 3.6).  I suppose the one time that won't be true is when we're updating to a newer python — but I don't know if we'd ever need to run multiple versions.

Nice to have — would be more convenient if the python version is a config instead of hard-coded value in a github action workflow file.

Reply:
sounds good! is there a single source of truth for the production version of python that each project targets? could try pulling in a config file.